### ISSUE TITLE
[HW3]Is it okay for me to try new answer to hw3?

### ISSUE THREAD

**Issue by GiWoongLee** [snu-sf-class/pp201701#33]
While I am studying pp for mid-term exams, I found out there has been some errors on my code so that I lost some point with hw3. I want to try my new code and get the result, not expecting better grade for homework. Just want to check whether new code is better than the prev ones.

Would it be okay? Thanks

Woong

**Comment by kim-yoonseung** [comment]
Yes. All submissions after the deadline is automatically filtered out from the actual grade.
